.github/workflows/ci.yml

```diff
@@ -64,6 +64,8 @@ jobs:
           - ember-canary
           - ember-default-with-jquery
           - ember-classic
+          - ember-concurrency-1.x
+          - ember-concurrency-2.x
```

Comment:
Do we really need two separate scenarios here? Like all the other scenarios are already using one of the versions, so we really just need one additional scenario to test with the other version, don't we?

Our test matrix is already quite big, and in this case two additional scenarios cause 4 additional test jobs, as each is running both for Bootstrap 3 and 4.

Reply:
Not really, but it does protect against losing coverage if the version in `yarn.lock` changes. I can limit it to just the `ember-concurrency-2.x` add, though.

Reply:
I think we should enforce a default version explicitly using yarn resolutions and only cover the other one with an explicit scenario.

I would be fine with using v2 as the default one starting from now. Than we can drop support for v1 in the next major.

Reply:
With the current version specifier in package.json, e-c 2.0.0 *would* get used by default if released, but not 2.0.0-beta.x, right? Or at least we need to update `yarn.lock`.

So can we just wait for the stable release, and then drop the `ember-concurrency-2.x` scenario?

Reply:
I'll keep the resolved version pinned at 1.x and remove the specific test setting for 1.x, and leave it up to your discretion on when to switch to 2.x